### DISCUSSION TITLE
1.4.7

### DISCUSSION THREAD

**Discussion by aviraxp** [LSPosed/LSPosed#795] [New Release]
### Changelog
- Fix system injection failure on some devices

### 更新日志
- 修复某些设备上的系统框架注入失败

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.4.7'>1.4.7</a>.</em>

**Comment by zy66cf66** [comment]
66666

**Comment by wjj01160309** [comment]
在哪下载

**Comment by huaihuaiw** [comment]
google 的pixel2通过magisk安装后,不显示lsposed manager

**Comment by Mister-Hope** [comment]
![Screenshot_20210808-182539](https://user-images.githubusercontent.com/33315834/128628914-6c2b5cee-bf83-4497-8b07-c67cf7632d5b.jpg)
大家好，我刚开始尝试这个项目，我已经从magisk仓库中安装了最新的1.4.7，为何还是出现需要新版的提示？

**Reply by ghost** [reply]
Because the app must check on internet if the latest version and until it can't access, tell you is not last version, I think is a little bug.
I had the same issue until the manager access to internet.

**Comment by Tittentoni** [comment]
Is IT for redmi 9 too?
